src/network/tcp.py

```diff
@@ -231,37 +242,55 @@ def handle_close(self):
 
 
 class Socks5BMConnection(Socks5Connection, TCPConnection):
+    """TBC"""
+
     def __init__(self, address):
         Socks5Connection.__init__(self, address=address)
         TCPConnection.__init__(self, address=address, sock=self.socket)
         self.set_state("init")
 
     def state_proxy_handshake_done(self):
+        """TBC"""
```

Comment:
State when SOCKS5 connection succeeds, we need to send a Bitmessage handshake to peer.